reVX/config/transmission_layer_creation.py

```diff
@@ -44,20 +45,37 @@ class Rasterize(BaseModel, extra='forbid'):
     """Reproject vector to raster CRS if ``True``"""
 
 
-class FBLayerConfig(BaseModel, extra='forbid'):
+class LayerBuildConfig(BaseModel, extra='forbid'):
     """
-    Friction and barrier layers config model. 'extent' is mandatory. 'map',
-    'range', and 'rasterize', and 'forced_inclusion' are exclusive, but one
-    must be specified.  Example configs can be seen in
-    test_xmission_barrier_friction_builder.py in the tests directory.
+    Friction and barrier layers config model.
+    'global_value', 'map', 'bins', 'rasterize', and 'forced_inclusion'
+    are exclusive, but exactly one must be specified. Example configs
+    can be seen in `test_xmission_barrier_friction_builder.py` in the
+    tests directory.
     """
-    extent: Extents
-    """Extent to apply map or range to"""
+
+    extent: Optional[Extents] = ALL
```

Comment:
I believe `Optional` implies `None` is an option. This looks like it's just `Extents` with a default

Reply:
Oops, didn't know that! Will fix.